From 65a3d47b0164b3afb905d73f80693659aa673b05 Mon Sep 17 00:00:00 2001
From: Klaus Kurzmann <mok@fluxnetz.de>
Date: Sat, 13 Dec 2008 10:31:03 +0100
Subject: [PATCH] make initscripts configurable

this makes it possible for distros to specify their
own initscripts package by setting DISTRO_INITSCRIPTS.

Author: Chia-I Wu <olv@openmoko.com>
Signed-off-by: Klaus Kurzmann <mok@fluxnetz.de>
---
 recipes/tasks/task-boot.bb |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/recipes/tasks/task-boot.bb b/recipes/tasks/task-boot.bb
index 3f31185..57315c9 100644
--- a/recipes/tasks/task-boot.bb
+++ b/recipes/tasks/task-boot.bb
@@ -1,5 +1,5 @@
 DESCRIPTION = "Basic task to get a device booting"
-PR = "r48"
+PR = "r48.1"
 
 inherit task
 
@@ -16,6 +16,7 @@ DISTRO_DEV_MANAGER ?= "${@base_contains("MACHINE_FEATURES", "kernel26",  "udev",
 # sysvinit, upstart
 #
 DISTRO_INIT_MANAGER ?= "sysvinit sysvinit-pidof"
+DISTRO_INITSCRIPTS ?= "initscripts"
 
 #
 # tinylogin, getty
@@ -39,7 +40,7 @@ RDEPENDS_task-boot = "\
     base-files \
     base-passwd \
     busybox \
-    initscripts \
+    ${DISTRO_INITSCRIPTS} \
     ${@base_contains("MACHINE_FEATURES", "keyboard", "keymaps", "", d)} \
     modutils-initscripts \
     netbase \
-- 
1.6.0.6

